protocol/app/flags/flags.go

```diff
@@ -50,8 +55,11 @@ const (
 	DefaultNonValidatingFullNode = false
 	DefaultDdErrorTrackingFormat = false
 
-	DefaultGrpcStreamingEnabled = false
-	DefaultVEOracleEnabled      = true
+	DefaultGrpcStreamingEnabled         = false
+	DefaultGrpcStreamingFlushIntervalMs = 50
+	DefaultGrpcStreamingMaxBufferSize   = 10000
```

Comment:
is 10k reasonable here? should we go higher in case of spikes in traffic?

Reply:
i guess if we are doing 10ms, 10k should be enough

Reply:
i'll use these for now and then tune later based on metrics